libs/permission-checker/tests/Check/Notification/CanModifySubscriptionsTest.php

```diff
@@ -67,22 +82,40 @@ public static function provideInvalidPermissionsCheckData(): iterable
             'error' => new PermissionDeniedException('Role "developer" is not allowed to modify subscriptions'),
         ];
 
+        yield 'developer role cross-branch' => [
+            'branchType' => null,
+            'token' => new StorageApiToken(features: ['protected-default-branch'], role: 'developer'),
+            'error' => new PermissionDeniedException('Role "developer" is not allowed to modify subscriptions'),
+        ];
+
         yield 'reviewer role on protected default branch' => [
             'branchType' => BranchType::DEFAULT,
             'token' => new StorageApiToken(features: ['protected-default-branch'], role: 'reviewer'),
             'error' => new PermissionDeniedException('Role "reviewer" is not allowed to modify subscriptions'),
         ];
 
+        yield 'reviewer role cross-branch' => [
+            'branchType' => null,
```

Comment:
A potřebujem vůbec crossbranch notifikace?

Reply:
:shrug: jako klasicky (ne-SOX) branches ted tak funguji, protoze v brnach i produkci mas stejny ID konfigurace a zadny filtr na branch

Reply:
ono to technicky neni cross-branch, spis branch-agnostic no, mozna jsem to jen blbe pojmenoval :slightly_smiling_face:

Reply:
aha, jasně. Čili je to spíš BWD kompatibility, než že by to bylo cross branch

Reply:
jj, tak neco. Jeste jsem to teda trochu prejmenoval, aby tam neblo to "cross-branch"